test/lib/analyzer/image-inspector.test.ts

```diff
@@ -84,4 +84,28 @@ test("get image as an archive", async (t) => {
     t.true(dockerPullSpy.calledOnce, "image pulled from remote registry");
     t.true(fs.existsSync(archiveLocation), "image exists on disks");
   });
+
+  t.test("from remote registry with authentication", async (t) => {
+    const dockerPullSpy: sinon.SinonSpy = sinon.spy(Docker.prototype, "pull");
+    const targetImage = "snykgoof/dockerhub-goof:alpine";
+
+    const username = process.env.DOCKER_HUB_USERNAME;
+    const password = process.env.DOCKER_HUB_PASSWORD;
```

Comment:
Should we improve the README to say this is now a requirement in order to run the test suite?

Reply:
Definitely, this is on my todo list. I expected this PR to change still as it's a WIP.

Reply:
@shaimendel in case this was missed!